cypress/e2e/22-stickies.cy.ts

```diff
@@ -0,0 +1,343 @@
+import { WorkflowPage as WorkflowPageClass } from '../pages/workflow';
+
+const workflowPage = new WorkflowPageClass();
+
+describe('Canvas Actions', () => {
+	beforeEach(() => {
+		cy.resetAll();
+		cy.skipSetup();
+		workflowPage.actions.visit();
+		cy.waitForLoad();
```

Comment:
There's no need to `waitForLoad` here, it's already part of the `workflowPage.actions.visit`